dags.yaml

```
@@ -126,6 +126,23 @@ bqetl_accounts_backend_external:
     - impact/tier_3
     - repo/bigquery-etl
 
+bqetl_accounts_derived:
+  schedule_interval: 30 2 * * *
+  description: |
+    Derived tables for analyzing data from Mozilla Accounts (`accounts_backend` and
+    `accounts_frontend` Glean applications).
+
+    This DAG is under active development.
+  default_args:
+    owner: akomar@mozilla.com
+    start_date: "2024-02-01"
```

```suggestion
    start_date: "2024-01-01"
```
Activity events landed on `2023-11-29` so technically `2023-11-30` is the earliest we can get here.